controller/kubernetes_configmap_controller.go

```diff
@@ -32,8 +32,8 @@ const (
 type KubernetesConfigMapController struct {
 	*baseController
 
+	namespace string
 	// use as the OwnerID of the controller
```

Comment:
just remove the comment, doesn't really add anything :)

Reply:
Thank you. Fixed